docs/platforms/dotnet/common/metrics/index.mdx

```diff
@@ -14,6 +12,8 @@ Metrics for .NET are supported with Sentry .NET SDK version `4.0.0` and above.
 
 Sentry metrics help you pinpoint and solve issues that impact user experience and app performance by measuring the data points that are important to you. You can track things like processing time, event size, user signups, and conversion rates, then correlate them back to tracing data in order to get deeper insights and solve issues faster.
 
+# Custom metrics
```

Comment:
```suggestion
# Custom Metrics

Sentry comes with built-in metrics, such as performance metrics and web vitals. You can also send custom metrics to Sentry to track the data points you care about most. This page describes how to set up and send custom metrics to use with our [Metrics feature](/product/metrics/). 
```

Wasn't sure if you meant to add more here as it's still a draft, but I think including an intro paragraph here to contextualize the rest of the info for this page would help.